eng/Version.Details.xml

```diff
@@ -37,6 +37,11 @@
       <Uri>https://github.com/dotnet/templating</Uri>
       <Sha>096be35db97669a59b298168b2be30f9b43eab8a</Sha>
     </Dependency>
+    <!-- this should have CPD to Roslyn but Roslyn doesn't have the proper Version.Details.xml entry - XXX issue number here -->
```

Comment:
Fill in?

> XXX issue number here

Reply:
Logged https://github.com/dotnet/runtime/issues/42637 and updated the comment.